tasks/main.yml

```diff
@@ -106,6 +106,33 @@
     - nextcloud_apps is mapping
   tags: install_apps
 
+- name: Disable Nextcloud Apps
```

Comment:
I think it is not needed to add this session to `main.yml`. A more compact and easy to maintain solution would be to add to the end of `nc_installation.yml` a block that executes `when: nextcloud_disable_apps is defined` to run `php occ app:disable "{{ nc_app_name }}"` with items from this list.

Reply:
@staticdev suggestion is even simpler than what I had i mind, so I :+1:  it

1tasks looping on the list and changed status depending of the stdout:
- ok : "No such app enabled: {{ item }}"
- changed: "{{ item }} <a version> enabled"

rc is always 0 in both case

should do something like that:
```yaml
changed_when:
  - "'enabled' in stdout"
  - "'No such app" not in stdout
```